doc/rate_limits.md

```diff
@@ -1,23 +1,31 @@
 ## Rate Limit API
 [Back to the navigation](README.md)
 
-Get Rate Limit
-Wraps [GitHub Rate Limit API](http://developer.github.com/v3/rate_limit/).
+Get rate limit wrappers from [GitHub Rate Limit API](http://developer.github.com/v3/rate_limit/).
 
-#### Get All Rate Limits.
+#### Get All Rate Limits
 
 ```php
-$rateLimits = $client->api('rate_limit')->getRateLimits();
+/** @var \Github\Api\RateLimit\RateLimitResource[] $rateLimits
+$rateLimits = $client->api('rate_limit')->getLimits();
 ```
 
 #### Get Core Rate Limit
 
 ```php
-$coreLimit = $client->api('rate_limit')->getCoreLimit();
+$coreLimit = $client->api('rate_limit')->getResource('core')->getLimit();
+$remaining = $client->api('rate_limit')->getResource('core')->getRemaining();
+$reset = $client->api('rate_limit')->getResource('core')->getReset();
 ```
 
 #### Get Search Rate Limit
 
 ```php
-$searchLimit = $client->api('rate_limit')->getSearchLimit();
+$searchLimit = $client->api('rate_limit')->getResource('search')->getLimit();
+```
+
+#### Get GraphQL Rate Limit
+
+```php
+$searchLimit = $client->api('rate_limit')->getResource('graphql')->getLimit();
```

Comment:
Is there an resource for graphql included in the rest api result? I can't find it in [the docs](https://developer.github.com/v3/rate_limit/#understanding-your-rate-limit-status)

Reply:
Yep, you're right it's not in there - not sure why. It is in the results of the existing API result from using this SDK though.

Reply:
Yes indeed I've just checked it with a curl call. So let's keep it in the docs!